client/src/components/SearchBar.js

```diff
@@ -1,32 +1,31 @@
-import React from "react";
-import { Input, Button } from "semantic-ui-react";
+import React, { useState } from 'react';
+import { Input, Button } from 'semantic-ui-react';
 
-class SearchBar extends React.Component {
-  static defaultProps = {
```

Comment:
removing default props should be okay here... we're passing in `loading` all the time so it will have a value of either `true` or `false`